src/util/geojson.js

```diff
@@ -86,17 +86,15 @@ export const buildEventCoordinateGetter = (headers, eventCoordinateField) => {
 
 export const createEventFeatures = (response, config = {}) => {
     const names = {
-        ...(config.outputIdScheme !== 'ID'
-            ? response.headers.reduce(
-                  (names, header) => ({
-                      ...names,
-                      [header.name]: header.column,
-                  }),
-                  {}
-              )
-            : null),
+        ...response.headers.reduce(
```

Comment:
Looking at this again I think we can actually leave out the `...` object spread operator now - it should just be:

```
const names = response.headers.reduce(
  ...
);